index.js

```diff
@@ -0,0 +1,5 @@
+// const DocumentCollection = require('./lib/document-collection');
+
+// const documents = new DocumentCollection(/* path to use */);
```

Comment:
index.js was set up for you to actually try running your methods and seeing if they actually write and read files/directories. The lab instructions specified that you should have done that here and console.logged what's returned from each. I'm not going to take points off, but I def recommend actually using the methods you wrote! Integration testing is an important supplement to Jest/mocking functions.